recipe/meta.yaml

```diff
@@ -13,7 +13,7 @@ source:
   {{ hash_type }}: {{ hash_val }}
 
 build:
-  number: 0
+  number: 1
```

Comment:
Since there was no PY36 a build `0` would be created for it. This  bumping will create redundant packages b/c  `PY<36 ` builds `0` and `1` will be the same.

Anyway, it does not hurt and that is why I merged. But the bump here was not needed.